### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by cristianarce89 — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]

File: components/organisms/EditAdmin/EditAdmin.module.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+.changePasswordUser-returnProfile:hover {
+  color: white;
+  cursor: pointer;
+}
+
+.aditAdmin__container--admins {
+  margin: 0 auto;
+  width: 600px;
```
Comment (DanielCalleCO) [inline components/organisms/EditAdmin/EditAdmin.module.css:21]: PX

File: components/organisms/EditAdmin/EditAdmin.module.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  height: calc(100% - 60px);
```
Comment (DanielCalleCO) [inline components/organisms/EditAdmin/EditAdmin.module.css:22]: PX

File: src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.content__editAdmin {
+  position: relative;
+  width: 800px;
```
Comment (DanielCalleCO) [inline src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css:11]: PX

File: src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  height: 500px;
```
Comment (DanielCalleCO) [inline src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css:12]: PX

File: src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  border-radius: 12px;
```
Comment (DanielCalleCO) [inline src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css:13]: PX

File: src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  background-color: var(--gray);
+}
+
+.toastContainer{
+  position: absolute;
+  top: 10px;
```
Comment (DanielCalleCO) [inline src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css:19]: PX

File: src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  right: 10px;
```
Comment (DanielCalleCO) [inline src/app/(privateRoutes)/dashboard/adminPanel/editAdmin/layout.module.css:20]: PX

File: components/organisms/EditAdmin/EditAdmin.tsx [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+              "Rol de administrador actualizado exitosamente",
+            "success"
+          );
+          dispatch(fetchAdmins(id));
+
+          router.push("/dashboard/adminPanel/editAdmin");
+        } else {
+          Swal.fire(
+            "Error!",
+            response.data.message || "Administrador no eliminado correctamente",
```
Comment (DanielCalleCO) [inline components/organisms/EditAdmin/EditAdmin.tsx:61]: This message does not make sense to me… this admin wasn't eliminated successfully?

File: services/user/allStudents.ts [hunk 9]
```diff
@@ -11,7 +11,7 @@ export const fetAllStudents = async (userId: string) => {
     );
     if (response.ok) {
       const dataUser = await response.json();
-
+
```
Comment (DanielCalleCO) [inline services/user/allStudents.ts:14]: delete this line

**Pull request merged**